exec/java-exec/src/test/java/org/apache/drill/exec/store/store/TestDisabledPlugin.java

```diff
@@ -58,10 +56,10 @@ public static void restore() throws Exception {
   public void testDisabledPluginQuery() throws Exception {
     try {
       run("SELECT * FROM cp.`employee.json` LIMIT 10");
-      fail("Query should have failed!");
+      Assert.fail("Query should have failed!");
```

Comment:
In Drill unit tests we usually use static import for assert methods. Please revert.